### ISSUE TITLE
Initializer for conditional binding must have Optional type, not 'String'

### ISSUE THREAD

**Issue by dachinat** [yannickl/QRCodeReader.swift#154]
Hello and thanks for this project.

I'm using  `pod 'QRCodeReader.swift', '~> 8.2.0'` version of your pod.

When trying to compile I get `Initializer for conditional binding must have Optional type, not 'String'`

Could you please help if you can?

<img width="1006" alt="screen shot 2018-09-28 at 9 13 28 pm" src="https://user-images.githubusercontent.com/22872749/46223119-80824600-c363-11e8-8870-acd01b0d9941.png">

**Comment by lixuansheng** [comment]
have you Solve it?

**Comment by lixuansheng** [comment]
I have the same problem.

**Comment by dachinat** [comment]
@lixuansheng yes, but I really don't remember what I did; I think there's example project somewhere around this repo, I managed to copy paste things and it started to work.
